tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -20277,8 +20277,10 @@ TEST_F(EnergyPlusFixture, DISABLED_AirLoopNumTest)
     ZoneAirLoopEquipmentManager::GetZoneAirLoopEquipment(*state);
     SimAirServingZones::GetAirPathData(*state);
 
+    // TODO: you can't go ahead and read them twice, or it'll just try to add the same object twice to the unordered_map solver.elements
+    // except we throw a fatal if that happens
     // Read AirflowNetwork inputs
-    state->afn->get_input();
+    // state->afn->get_input();
```

Comment:
This should just be deleted, it may have once been possible (but not a good idea) to read the input twice, but it is not now (and still not a good idea).